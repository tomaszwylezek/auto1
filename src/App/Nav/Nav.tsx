import React, { FC } from 'react';
import { Link } from 'react-router-dom';

export const Nav: FC = (): JSX.Element => (
  <nav className="navbar navbar-light bg-light">
    <Link className="navbar-brand" to="/">
      Auto1 - JS Coding test
    </Link>
  </nav>
);
