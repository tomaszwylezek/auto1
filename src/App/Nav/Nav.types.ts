export interface INavOwnProps {}

export interface INavConnectedState {}

export interface INavConnectedActions {}

export interface INavProps extends INavConnectedState, INavConnectedActions, INavOwnProps {}


