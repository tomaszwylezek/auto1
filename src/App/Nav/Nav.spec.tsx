import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import { Nav } from './Nav';

// Models
import { INavProps } from './Nav.types';

describe('Nav component should', (): void => {
  let props: INavProps;
  let component: ShallowWrapper<INavProps>;

  beforeEach((): void => {
    props = {};

    component = shallow(<Nav {...props} />);
  });

  it('render correctly', (): void => {
    expect(component).toMatchSnapshot();
  });
});
