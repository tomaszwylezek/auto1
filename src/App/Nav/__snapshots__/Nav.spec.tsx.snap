// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component should render correctly 1`] = `
<nav
  className="navbar navbar-light bg-light"
>
  <Link
    className="navbar-brand"
    to="#"
  >
    Auto1 - JS Coding test
  </Link>
</nav>
`;
